src/Compilers/CSharp/Portable/Binder/ExpressionVariableFinder.cs

```diff
@@ -52,24 +61,30 @@ internal class ExpressionVariableFinder : CSharpSyntaxWalker
 
             var finder = s_poolInstance.Allocate();
             finder._scopeBinder = binder;
+            finder._enclosingBinder = binder;
             finder._localsBuilder = builder;
 
             foreach (var n in nodes)
             {
-                finder.Visit(n);
+                finder._nodeToBind = n;
```

Comment:
No.

---
In reply to: [76321466](https://github.com/dotnet/roslyn/pull/13274#discussion_r76321466) [](ancestors = 76321466)